stepsfunction/handler.py

```diff
@@ -1,23 +1,29 @@
 """Simple example."""
 from json import dumps, loads
 import io
-
+import sys
 import boto3
+import botocore
+from botocore.exceptions import ClientError
 import logging
 import os
-from models import PDFUpload
+from models import PDFUpload, SinglePage
```

Comment:
I want to work with you to use define a modern data model for the DynamoDB. These 'models' abstractions make it work like SQL almost, and will prevent us from using DDB in the fastest, cheapest way possible.  This is fine for now, so just keep going.